native/core/Cargo.toml

```diff
@@ -52,6 +52,8 @@ serde = { version = "1", features = ["derive"] }
 lazy_static = "1.4.0"
 prost = "0.12.1"
 jni = "0.21"
+snap = "1.1"
+lz4_flex = { version = "0.11.3", default-features = false }
```

Comment:
This is a somewhat confusingly named feature flag. Can we leave a comment here that we're enabling unsafe encode and decode for performance?